Makefile

```diff
@@ -48,7 +51,7 @@ $(COV_REPORT): $(PKG_FILES) $(ALL_SRC)
 		-ignore "$(OVERALLS_IGNORE)" \
 		-covermode=atomic \
 		$(DEBUG_FLAG) -- \
-		$(TEST_FLAGS) $(RACE) $(TEST_VERBOSITY_FLAG) | \
+		$(TEST_FLAGS) $(RACE) -timeout=1s $(TEST_VERBOSITY_FLAG) | \
```

Comment:
lets move timeout into an actual flag `$(TEST_TIMEOUT)` since it's used in more than one place.